internal_transfers/actions/src/artifacts/GPv2Settlement.json

```diff
@@ -0,0 +1,590 @@
+{
+  "_format": "hh-sol-artifact-1",
+  "contractName": "GPv2Settlement",
+  "sourceName": "src/contracts/GPv2Settlement.sol",
+  "abi": [
+    {
+      "inputs": [
+        {
+          "internalType": "contract GPv2Authentication",
+          "name": "authenticator_",
+          "type": "address"
+        },
+        {
+          "internalType": "contract IVault",
+          "name": "vault_",
+          "type": "address"
+        }
+      ],
+      "stateMutability": "nonpayable",
+      "type": "constructor"
+    },
```

Comment:
I have not checked whether this is already a trimmed version or not, but one thing you can do to reduce the amount of unnecessary info added to the repo is to pick exactly what you need to use and nothing else.

For example, do you need `simulateDelegatecall`?
And I've never seen the bytecode added before, you might not need this either.

Reply:
You are totally right, I just copied the entire file from the repo -- there are, in fact, very few parts of the ABI we actually need. I can substantially reduce this.